merlin_ai/ai_classes.py

```diff
@@ -88,6 +89,9 @@ def __call__(
         :param return_raw_response: if True raw response from LLM API will be returned alongside the AIModel instance
         :return: instance of data class
         """
+        if not isinstance(self._model_config, ModelFromDataClass):
+            raise Exception("This model is not callable.")
```

Comment:
How do we handle this is the model was not initialized from a DataClass? Do we populate a dict here?